apps/modernization-ui/src/apps/search/investigation/investigationTermsResolver.ts

```diff
@@ -25,26 +25,42 @@ const processingStatus = fromSelectables('processingStatuses', 'PROCESSING STATU
 const notificationStatus = fromSelectables('notificationStatuses', 'NOTIFICATION STATUS');
 
 const investigationTermsResolver = (entry: InvestigationFilterEntry): Term[] => {
+    if (!entry) {
+        return [];
+    }
     const terms: Term[] = [];
 
-    conditions(entry.conditions).forEach((item) => terms.push(item));
-    programAreas(entry.programAreas).forEach((item) => terms.push(item));
+    if (entry.conditions) {
```

Comment:
I think we dont need these if conditions here either,, `conditions(entry.conditions).forEach((item) => terms.push(item));`

should be good enough